src/CollectionCardMaker.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CollectionCardMaker {
+    private string $name;
+    private string $playercount;
```

Comment:
everything (almost) in OOP should be camelCase - so this should be called $playerCount